src/main/features/response/views/task-list.njk

```diff
@@ -55,12 +55,10 @@
           <li>
             <strong class="bold">{{ t('Total claim amount:') }}</strong>
             <span class="block">{{ claim.totalAmountTillToday | numeral }}</span>
-            <div>{{ t('<a href="%s">View claim</a>',
-                {
-                  postProcess: 'sprintf',
-                  sprintf: [ResponsePaths.claimDetailsPage.evaluateUri({ externalId: claim.externalId })]
-                }
-              ) | safe }}</div>
+          </li>
+          <li>
+            <strong class="bold">{{ t('Claim details:') }}</strong>
+            <span class="block"><a href="{{ ResponsePaths.claimDetailsPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('View claim') }}</a></span>
```

Comment:
have you tried using a div? it is a block level element and then you shouldn't need the class

Reply:
Done